Editor/ReactiveObjects/ParameterAssignerPass.cs

```diff
@@ -186,6 +187,19 @@ protected override void Execute(ndmf.BuildContext context)
 
                 expParams.parameters = expParams.parameters.Concat(newParameters.Values).ToArray();
             }
+
+            if (_mamiByParam.Count > 0)
+            {
+                // This make sures the parameters are correctly merged into the FX layer.
+                context.AvatarRootObject.AddComponent<ModularAvatarMergeAnimator>().animator = new AnimatorController
+                {
+                    parameters = _mamiByParam.Keys.Select(name => new AnimatorControllerParameter
+                    {
+                        name = name,
+                        type = AnimatorControllerParameterType.Float,
+                    }).ToArray(),
+                };
```

Comment:
As a best practice, please be specific about relevant configuration (e.g., target animator layer = FX, remove attached animator = false)